js index.babel

```diff
@@ -24,4 +24,5 @@ INSTRUCTIONS:
 
 // PLEASE NOTE: Adding global style rules using the * selector, or by adding rules to body {..} or html {..}, or to all elements within body or html, i.e. h1 {..}, has the potential to pollute the test suite's CSS. Try adding: * { color: red }, for a quick example!
 
+
 // Once you have read the above messages, you can delete all comments. 
```

Comment:
I think it would be better if you remove unused comments in your code. Unused codes make your code unclean. Thanks.